app/controllers/schools_controller.rb

```diff
@@ -11,6 +11,11 @@ def new
   end
 
   def create
+    
+    unless params[:school][:accepts_financial_aid]
+      params[:school][:accepts_financial_aid] = false
+    end
+
```

Comment:
I like this a lot... Your use of a single checkbox rather than a selection of true or false is awesome. Nice work!!